status-im/status-react
# Add ios security framework

Fixes: #7226 

Adds sqlcipher to cocoa pods for ios.

It also upgrades status-go with a migration, after this IOS devices might have slower login.

To test:
1) Install previous version on ios (0.9.32)
2) Install same version on a different device and pair the devices.
3) Make sure they are paired.
4) Upgrade ios to this pr
5) Make sure they are still paired ok

Includes kdf changes in status-go can exclude if necessary.
status: ready
